test/parallel/test-fs-mkdir.js

```diff
@@ -71,6 +72,79 @@ tmpdir.refresh();
   );
 });
 
+// mkdirpSync when folder does not yet exist.
+{
+  const pathname = `${tmpdir.path}/test4/test5`;
+
+  fs.mkdirSync(pathname, { parent: true });
+
+  const exists = common.fileExists(pathname);
+  assert.strictEqual(exists, true);
+}
+
+// mkdirpSync when folder already exists.
+{
+  const pathname = `${tmpdir.path}/test6/test7`;
+
+  fs.mkdirSync(pathname, { parent: true });
+  // should not cause an error.
+  fs.mkdirSync(pathname, { parent: true });
+
+  const exists = common.fileExists(pathname);
+  assert.strictEqual(exists, true);
+}
+
+// mkdirpSync ../
+{
+  const pathname = `${tmpdir.path}/../`;
```

Comment:
Should this be something like ${tmpdir.path}/testX/testY/../

or even

${tmpdir.path}/testX/../testX/testY/

to make sure the directory created is under the temporary directory that gets cleaned up?


so that we know its creating